frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -302,8 +302,6 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                         <FeatureFlagReleaseConditions />
                         {featureFlags[FEATURE_FLAGS.FF_CODE_EXAMPLE] && (
                             <>
-                                <LemonDivider />
-                                <FeatureFlagCodeExample featureFlag={featureFlag} />
                                 <Field name="ensure_experience_continuity">
```

Comment:
also bumped the "persist feature flag" checkbox higher so it's above the code example and more noticeable?

Reply:
Like where it was originally? Or where it was before I bumped it..? Because for 2), that's because you asked for it to be bumped higher which made sense to me and for 1), because persisting auth is tied to implementation details... no? It also makes sense for the user to not have to scroll up and down and back and forth so much between that checkbox and the information for it along with local evaluation or bootstrapping

Reply:
I meant originally yes (that's what I meant for it to be bumped back up).

It's not tied to implementation details (even though it affects implementation). The main thing is a business use case: That when users want a certain behaviour, they check it. Hence wanting it to be close to the top where it's next to other config options, like enabling the flag.

Now based on this behaviour, what they can do with the implementation changes.

Reply:
Could be wrong, but also, people aren't really using that box with where it's at right now, right? So why don't we try experimenting with what's not working at least?

Reply:
Implementation is _how_ they do it. It's less important than _what_ they want to do. The purpose is not related to implementation hence why it doesn't make sense for it to go there, to me.

Persist flag across auth steps is the "what". How they do it is an implementation detail that doesn't affect the business use case.

Reply:
updated previous comment to clarify the purpose, the behaviour is not _just_ checking it, it's what checking it means: which is that identify events will not flip-flop your flag variants

Reply:
Q: How is the implementation related to this configuration?

Reply:
> , people aren't really using that box with where it's at right now, right?

Not true from what I've seen. People who want it, use it. Most people don't want it, which is a separate thing though.